ale/isd_generate.py

```diff
@@ -143,6 +160,15 @@ def file_to_isd(
     else:
         usgscsm_str = ale.loads(file)
 
+    if radius is not None:
+        usgscsm_json = json.loads(usgscsm_str)
+        usgscsm_json["radii"]["semimajor"] = radius
+        usgscsm_json["radii"]["semiminor"] = radius
+        logger.info(f"Overriding radius to:")
+        logger.info(usgscsm_json["radii"])
+        usgscsm_str = json.dumps(usgscsm_json, indent=2)
+        isd_file.write_text(usgscsm_str)
```

Comment:
Remove line 170, otherwise you'll write the isd_file twice.

Reply:
good catch